poutyne/framework/metrics/epoch_metrics/fscores.py

```diff
@@ -0,0 +1,259 @@
+"""
```

Comment:
We also should add the F1Score. Since most of the time we use the F metric with a beta of 1, it's convenient (also for the lambda user) to juste have a F1 score .

Reply:
It's already there.

Reply:
Haven't see it, my bad.